environment.yml

```diff
@@ -0,0 +1,90 @@
+name: teamfight38
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - _py-xgboost-mutex=2.0=cpu_0
```

Comment:
dont think we need to list all the sub dependencies just do like:
 - python=3.8
 - numpy=X
 - pandas
 - etc

like just list the main installs then it will sort out the rest, bc if one of these mini subdependencies change whole thing wont work